api/src/controllers/topics.js

```diff
@@ -54,28 +60,35 @@ const githubFunc = async (
 ) => {
   let response = { statusMessage: 'Ok', prUrl: '', status: '200' };
   try {
-      // create new topic and branch
-      // validate topic schema
-      const isValidData = validate(req.body);
-      if (isValidData) {
-        // TODO -- validate topic doesn't already exist
-        // TODO -- validate topic sources are valid
-        // create a git branch on the remote with a naming convention [createTopic/topicname]
-        await createNewRefFromBase(owner, repo, ref);
-        // // create a new file with contents
-        await createFile(owner, repo, bodyData, ref, topicName, email, name);
-        // commit  to branch
-        // make pr against ref to base using templates
-        const pullRequest = await createPullRequest(owner, repo, defaultBranch, topicName, ref);
-        // URL to the pull Request created ..
-        const pullRequestUrl = pullRequest.data.html_url;
-        response.prUrl = pullRequestUrl;
-        //eslint-disable-next-line
+    // create new topic and branch
+    // validate topic schema
+    const isValidData = validate(req.body);
+    if (isValidData) {
+      // TODO -- validate topic doesn't already exist
```

Comment:
just make sure to create github issues for these TODO's so they are not missed. be descriptive. 

File name etc